server/athenian/api/experiments/preloading/entries.py

```diff
@@ -203,8 +204,26 @@ async def _fetch_prs_by_filters(
         return prs
 
 
-class PreloadedMetricEntriesCalculator(MetricEntriesCalculator):
+class PreloadedPullRequestJiraMapper(PullRequestJiraMapper):
+    """Mapper of pull requests to JIRA tickets."""
+
+    @classmethod
+    @sentry_span
+    async def load_pr_jira_mapping(cls,
+                                   prs: Collection[str],
+                                   meta_ids: Tuple[int, ...],
+                                   mdb: databases.Database) -> Dict[str, str]:
+        """Fetch the mapping from PR node IDs to JIRA issue IDs."""
+        cached_df = mdb.cache.dfs["jira_mapping"]
+        df = cached_df.df
+        mask = df["node_id"].isin([v.encode() for v in prs]) & df["node_acc"].isin(meta_ids)
+        mapping = cached_df.filter(mask)
+        return dict(zip(mapping["node_id"].to_list(), mapping["jira_id"].to_list()))
```

Comment:
```suggestion
        return dict(zip(mapping["node_id"].values, mapping["jira_id"].values))
```
`.values` is the fastest way to iterate Pandas column values, proven by extensive profiling.

Reply:
thanks!